File: templates/Virtual-Assistant-Template/typescript/generator-botbuilder-assistant/generators/skill/templates/customSkill/src/models/skillState.ts

/**
 * Copyright(c) Microsoft Corporation.All rights reserved.
 * Licensed under the MIT License.
 */
import { RecognizerResult } from 'botbuilder';

export class SkillState {
    public readonly token: string = '';
    public luisResult: RecognizerResult | undefined;
    // tslint:disable-next-line: no-empty
    public clear(): void {}
}

File: templates/Skill-Template/csharp/Template/Skill/Models/SkillState.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

using Luis;

namespace $safeprojectname$.Models
{
    public class SkillState
    {
        public string Token { get; set; }

        public $safeprojectname$Luis LuisResult { get; set; }

        public void Clear()
        {
        }
    }
}
